pkg/cosign/tuf/policy.go

```diff
@@ -66,15 +58,14 @@ func (k *Key) ContainsID(id string) bool {
 }
 
 type Root struct {
-	Type        string           `json:"_type"`
-	SpecVersion string           `json:"spec_version"`
-	Version     int              `json:"version"`
-	Expires     time.Time        `json:"expires"`
-	Keys        map[string]*Key  `json:"keys"`
-	Roles       map[string]*Role `json:"roles"`
-	Namespace   string           `json:"namespace"`
-
-	ConsistentSnapshot bool `json:"consistent_snapshot"`
```

Comment:
What's `ConsistentSnapshot`? Is it okay that we're removing it here?

Reply:
Yeah, it's TUF language for versioning the names of the policy, which we aren't doing yet anyway (e.g. naming the root policies 1.root, 2.root, 3.root).... unless we do want to do that now that I think about it...